### PR TITLE
Pagination for get all droplets

### PR DESCRIPTION
**Pull request by ghost**
@funkatron Please review.

### CODE REVIEW

File: src/Api/Droplet.php [hunk 1]
```diff
@@ -27,14 +27,24 @@ class Droplet extends AbstractApi
...
     {
-        $droplets = $this->adapter->get(sprintf('%s/droplets?per_page=%d', self::ENDPOINT, PHP_INT_MAX));
-        $droplets = json_decode($droplets);
-
-        $this->extractMeta($droplets);
-
-        return array_map(function ($droplet) {
+        $thereAreNewResults = true;
+        $results = [];
+        $url = sprintf('%s/droplets?per_page=%d', self::ENDPOINT, PHP_INT_MAX);
```
Comment (funkatron) [inline src/Api/Droplet.php]: I wonder if setting per_page to `PHP_INT_MAX` isn't kind of abusive of their API. Maybe something like 1000?

Comment (ghost) [inline src/Api/Droplet.php]: @funkatron I tried. They ignore any value > 200. As you can see, it is like this in the original module and it's not make any difference.

Comment (funkatron) [inline src/Api/Droplet.php]: Well, maybe they’ll support 1000 at some time. I’d stick the pagination
value in a class `const` and set it to 1000.

Comment (funkatron) [inline src/Api/Droplet.php]: Did this myself.

**Pull request merged**